src/metabase/db/custom_migrations.clj

```diff
@@ -1070,6 +1070,20 @@
                                            :from   [:revision]
                                            :where  [:= :model "Card"]})))))
 
+(define-migration DeleteScanFieldValuesTriggerForDBThatTurnItOff
+  (let [dbs-without-scan-field-values (filter #(and (-> % :details :let-user-control-scheduling)
```

Comment:
nit: This could be a `when-let` if you bring the seq up.

nit: The binding is quite a mouthful, yet still not that clear. How about just using a short name like `databases` or `db-ids` (see below), and a line comment like "Find the databases which are configured not to periodically scan and cache their field values"?